packages/amplify-util-uibuilder/src/commands/generateComponents.ts

```diff
@@ -50,7 +53,11 @@ export const run = async (context: $TSContext): Promise<void> => {
 
     const invalidComponentNames = componentSchemas.entities.filter(component => !component.schemaVersion).map(component => component.name);
     if (invalidComponentNames.length) {
-      printer.warn(`The components ${invalidComponentNames.join(', ')} were synced with an older version of Amplify Studio. Please re-sync your components with Figma to get latest features and changes.`);
+      printer.warn(
+        `The components ${invalidComponentNames.join(
```

Comment:
Is there any sort of localization in the CLI or is it only English?

Reply:
Not currently